.vscode/launch.json

```diff
@@ -9,14 +9,34 @@
       "request": "launch",
       "name": "Run Mocha tests",
       "program": "${workspaceRoot}/packages/build/node_modules/.bin/_mocha",
+      "runtimeArgs": ["-r", "${workspaceRoot}/packages/build/node_modules/source-map-support/register"],
       "cwd": "${workspaceRoot}",
       "args": [
         "--config",
         "${workspaceRoot}/packages/build/config/.mocharc.json",
         "packages/*/dist/__tests__/**/*.js",
         "-t",
         "0"
-      ]
+      ],
+      "windows": {
+        "program": "${workspaceRoot}/packages/build/node_modules/mocha/bin/mocha",
```

Comment:
I think this part is no longer needed, see https://github.com/strongloop/loopback-next/pull/4643/commits/992dfe98c4c10087ed30125e5e4faed734f8a890

Reply:
Done!

Reply:
I think we misunderstood each other. I was trying to say that the `windows` section is no longer needed, because the main `program` setting was already updated to work on all platforms (Windows, Linux, MacOS).

I _think_ that your current config will actually break debugging Mocha tests on Windows, because `mocha` is just a thin wrapper that spawns another child process (running `_mocha` script).

Anyhow, let's remove the `windows` section please, it shouldn't be needed anymore.

Reply:
Fixed!